htdocs/stripe/class/stripe.class.php

```diff
@@ -396,16 +398,19 @@ public function createPaymentStripe($amount, $currency, $origin, $item, $source,
 					$charge = \Stripe\Charge::create(array(
 						"amount" => "$stripeamount",
 						"currency" => "$currency",
-						// "statement_descriptor" => " ",
+                                                "capture"  => true,
+						"statement_descriptor" => dol_trunc(dol_trunc(dol_string_unaccent($customer->name), 6, 'right', 'UTF-8', 1).' '.$description, 22, 'right', 'UTF-8', 1),     // 22 chars that appears on bank receipt
```

Comment:
It is recommended by Stripe to add the seller business name so customer know who he paid. Otherwise, customer has no chance to know which companies he paid and he complain to its bank to be refunded, and this make Stripe to not refund moneys to seller.

So i suggest to put instead 
"statement_descriptor" => dol_trunc(dol_trunc(dol_string_unaccent($mysoc->name), 6, 'right', 'UTF-8', 1).' '.$description, 22, 'right', 'UTF-8', 1)

Reply:
done ;-)